integrations/aws/main.py

```diff
@@ -224,7 +225,19 @@ async def webhook(update: ResourceUpdate, response: Response) -> fastapi.Respons
                 resource = await describe_single_resource(
                     resource_type, identifier, account_id, region
                 )
-            except Exception:
+            except Exception as e:
+                if is_access_denied_exception(e) or is_server_error(e):
+                    logger.error(
+                        f"Skipping resyncing {resource_type} in region {region} due to missing access permissions"
+                    )
+                    return fastapi.Response(
+                        status_code=status.HTTP_200_OK,
+                        content=json.dumps({"ok": True}),
+                    )
+                resource = None
```

Comment:
if access denied we do want to try to delete I think

Reply:
Why is that?